scholia/app/templates/publisher.html

```diff
@@ -142,6 +174,16 @@ <h3 id="As function of number of published works">As function of number of publi
 </div>
 
 
+<h2>Retractions</h2>
+
+<id="Retractions">Retracted papers in this journal</h2>
+
+<table class="table table-hover" id="retracted"></table>
+
+<div class="embed-responsive embed-responsive-4by3">  
```

Comment:
@fehrhart, why are lines 183-185 with the `iframe` here? If I follow the URL, I just get the WDQS. What are you trying to do here?